recipes/su2/conda_build_config.yaml

```diff
@@ -0,0 +1,10 @@
+python:
```

Comment:
This file should not be necessary, rather you should add a `skip: True  # [py>=312]` to build

Reply:
I'll try again but it was failing with `skip`

Reply:
It works fine!
Thanks for the suggestion, @xhochy !